src/utils/constants/constants.ts

```diff
@@ -0,0 +1,4 @@
+/* GENERIC DIALOGUE:
+This will be used across help messages & intro */
+export const WELCOME_DIALOGUE = "Welcome to brsh, a lightweight browser shell!";
+export const HELP_SUBHEADER = "Try the \"help\" command for more detail about brsh";
```

Comment:
It might also be useful to print this out when the terminal is opened since users wouldn't know what they can run in the terminal.

Reply:
oh dang forgot to do this before i merged. will do in the next PR i release